api/scheduler_config_handler_test.go

```diff
@@ -67,13 +68,29 @@ var _ = Describe("SchedulerConfigHandler", func() {
 			Expect(yamlResp["name"]).To(Equal(configYaml.Name))
 		})
 
+		It("should return config json of current version", func() {
+			MockSelectYaml(yamlString, mockDb, nil)
+
+			request, _ = http.NewRequest("GET", url, nil)
+			request.SetBasicAuth(user, pass)
+			request.Header.Add("Accept", "application/json")
+			app.Router.ServeHTTP(recorder, request)
+			Expect(recorder.Code).To(Equal(http.StatusOK))
+
+			yamlBytes, _ := ghodssYaml.JSONToYAML(recorder.Body.Bytes())
+			var yamlResp map[string]interface{}
+			yaml.Unmarshal(yamlBytes, &yamlResp)
+			Expect(yamlResp["name"]).To(Equal(configYaml.Name))
+		})
+
 		It("should return config yaml of specified version", func() {
 			version := "v1.0"
 			MockSelectYamlWithVersion(yamlString, version, mockDb, nil)
 
 			url = fmt.Sprintf("http://%s/scheduler/%s/config?version=%s",
 				app.Address, configYaml.Name, version)
 			request, _ = http.NewRequest("GET", url, nil)
+			request.SetBasicAuth(user, pass)
```

Comment:
By removing on BeforeEach and adding this, you are removing the test with Oauth. 
Please, add a new test that uses oauth.